src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -67,25 +62,22 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
      */
     onYearChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
         const nextYear = parseInt(event.target.value);
-        let election = this.props.electionType.elections.find((election) => election.year === nextYear);
         let votes = this.props.votes.filter((vote) => vote.electionYear === nextYear);
         const distributionYear = this.props.use2021Distribution && nextYear >= 2005 ? 2021 : nextYear;
         let metrics = this.props.metrics.filter((metric) => metric.electionYear === distributionYear);
         const parameters =
             this.props.parameters.find((parameter) => parameter.electionYear === nextYear) || unloadedParameters;
 
-        if (election !== undefined) {
+        if (parameters !== undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5976811)